helm/loki/values.yaml

```diff
@@ -233,67 +252,17 @@ loki:
         memory: 3Gi
         cpu: 500m
 
-  loki:
-    image:
-      repository: giantswarm/loki
-    memcached:
```

Comment:
memcache configuration was moved to https://github.com/giantswarm/loki-app/pull/320/files#diff-ef20f7c6ed4d3ce4c7be2e6cf430ec1b31714304429f5a6ced9a046fe0abd1b0R256 so disabling by default (not sure if we need to though)

Reply:
Why enable it by default?
It requires to have memcache app running, which we won't support. So I'd rather let it be a conscious choice from customers if they want to enable it.

Reply:
So, new loki has 2 cache mechanisms.

The one under memcached is for external memcached which is now disabled by default.

The other one, under chunksCache and resultsCache is enavled by default and run a custom memcached. I would guess that if they advise to run it, we should let it be a default for customers to have it and they can disable it if they want